src/LCT.PackageIdentifier/NugetProcessor.cs

```diff
@@ -495,9 +449,33 @@ private static void ConvertToCycloneDXModel(List<Component> listComponentForBOM,
                     }
                 };
                 listComponentForBOM.Add(components);
+                if (prop.Dependencies != null)
+                {
+                    GetDependencyDetails(components, prop, ref dependencies);
+                }
             }
         }
 
+        private static void GetDependencyDetails(Component compnent, NugetPackage prop,ref List<Dependency> dependencies)
+        {
+            List<Dependency> subDependencies = new();
+            foreach (var item in prop?.Dependencies)
```

Comment:
i don't think Null check is required here ,Bcz while calling only u checked prop.Dependencies !=null

Reply:
removed